src/rii.h

```diff
@@ -206,15 +211,15 @@ std::pair<std::vector<size_t>, std::vector<float> > RiiCpp::QueryLinear(const py
         size_t N = GetN();
         scores.resize(N);
 #pragma omp parallel for
-        for (size_t n = 0; n < N; ++n) {
+        for (long long n = 0; n < N; ++n) {
```

Comment:
Do you need this conversion for windows?

Just in case, can you `static_cast` each variable like this? (It's messy though...)
https://github.com/DwangoMediaVillage/pqkmeans/blob/master/src/clustering/pqkmeans.cpp#L89-L90

(See the context here https://github.com/DwangoMediaVillage/pqkmeans/pull/4/files#diff-6dfef36c6a2e1d58af1876074c532d53224515a184caf3a114740a2d5e1be231R85-R86)